lms/envs/pact.py

```diff
@@ -0,0 +1,14 @@
+"""
+Settings for Pact Verification Tests.
+"""
+
+from .test import *  # pylint: disable=wildcard-import, unused-wildcard-import
+
+#### Allow Pact Provider States URL ####
+PROVIDER_STATES_URL = True
+
+#### Default User name for Pact Requests Authentication #####
+MOCK_USERNAME = 'Mock User'
```

Comment:
same comment for username update

Reply:
As discussed, this username exists in contract.